.github/workflows/check-advisories.yml

```diff
@@ -67,7 +67,7 @@ jobs:
             ID=$(basename "$FILE" .md)
             YEAR=$(echo "$ID" | cut -d - -f 2)
             mkdir -p $DATA_DIR/$YEAR
-            docker run --rm -v $PWD:/advisories haskell/hsec-tools:latest /bin/hsec-tools osv "$FILE" > $DATA_DIR/$YEAR/$ID.json
+            docker run --rm -v $PWD:/advisories haskell/hsec-tools:latest /bin/hsec-tools osv "advisories/$FILE" > $DATA_DIR/$YEAR/$ID.json
           done < <(find advisories -type f -name "*.md")
```

Comment:
And now I see the `$PWD:/advisories` which is a bit confusing...

Perhaps we should change the container volume mount point from `/advisories` to `/security-advisories` or even just `/repo`, to distinguish it from the `advisories` *subdirectory* and (hopefully?) make this code a bit easier to follow.

Reply:
I do like the `${{ github.workspace }}` change, but I was proposing to change the directory name on the *container* side :)